libs/product/src/facades/configurable-product/configurable-product-facade.interface.ts

```diff
@@ -31,12 +31,36 @@ export interface DaffConfigurableProductFacadeInterface extends DaffStoreFacade<
 	 */
 	getMaximumPrice(id: string): Observable<number>;
 
+	/**
+	 * Get the current maximum price possible based on the applied attributes and remaining variants.
+	 * @param id the id of the configurable product.
+	 */
+	getMaximumPrice(id: string): Observable<number>;
```

Comment:
This duplicates the above interface key.

Reply:
oops. removed